ngraph/core/include/ngraph/op/random_uniform.hpp

```diff
@@ -63,6 +63,8 @@ class NGRAPH_API RandomUniform : public Op {
         m_op_seed = seed2;
     }
 
+    bool evaluate(const HostTensorVector& outputs, const HostTensorVector& inputs) const override;
```

Comment:
add method has_evaluate(), by default this method return false

Reply:
Done.